cmd/oaas/handler_build_test.go

```diff
@@ -122,13 +124,27 @@ echo "fake-build-result" > %[1]s/build/output/disk.img
 	assert.True(t, stat.IsDir())
 
 	// now get the result
-	endpoint = baseURL + "api/v1/result/disk.img"
+	endpoint = baseURL + "api/v1/result/image/disk.img"
 	rsp, err = http.Get(endpoint)
 	assert.NoError(t, err)
 	assert.Equal(t, http.StatusOK, rsp.StatusCode)
 	body, err := ioutil.ReadAll(rsp.Body)
 	assert.NoError(t, err)
 	assert.Equal(t, "fake-build-result\n", string(body))
+
+	// check that the output tarball has the disk in it
+	endpoint = baseURL + "api/v1/result/output.tar"
+	rsp, err = http.Get(endpoint)
+	assert.NoError(t, err)
+	assert.Equal(t, http.StatusOK, rsp.StatusCode)
+	body, err = ioutil.ReadAll(rsp.Body)
+	assert.NoError(t, err)
+	tarPath := filepath.Join(baseBuildDir, "output.tar")
+	assert.NoError(t, os.WriteFile(tarPath, body, 0644))
+	cmd := exec.Command("tar", "-tf", tarPath)
+	out, err := cmd.Output()
+	assert.NoError(t, err)
+	assert.True(t, strings.HasSuffix(strings.Split(string(out), "\n")[2], "image/disk.img"))
```

Comment:
(nitpick) How bad/annoying would a full `assert.Equal(t, out, `./foo\nbar\n...`) here be? It might be too much/to many changes but maybe not :) This way we would know if we accidently include too much in a future refator.

Reply:
fair point